tests/test_browser.py

```diff
@@ -2539,6 +2540,27 @@ def test_webgl2_garbage_free_entrypoints(self):
   def test_webgl2_backwards_compatibility_emulation(self):
     self.btest(path_from_root('tests', 'webgl2_backwards_compatibility_emulation.cpp'), args=['-s', 'USE_WEBGL2=1', '-s', 'WEBGL2_BACKWARDS_COMPATIBILITY_EMULATION=1'], expected='0')
 
+  @requires_graphics_hardware
```

Comment:
does this test pass on all browsers? i thought none support this feature yet (without a flag)?

the test will have to be off by default if it doesn't pass on our CI.

Reply:
I didn't find proper function annotations for this, so it's explicitly skipped below.